do_cmake.sh

```diff
@@ -19,7 +19,9 @@ if [ -r /etc/os-release ]; then
   source /etc/os-release
   case "$ID" in
       fedora)
-          if [ "$VERSION_ID" -ge "37" ] ; then
+          if [ "$VERSION_ID" -ge "39" ] ; then
```

Comment:
it's not in the scope of this change. but how about dropping the branches for fedora releases which are already EOL? namely `$release` <  37 . see https://docs.fedoraproject.org/en-US/releases/eol/

Reply:
Hi @tchaikov , of course, that makes perfect sense. Please see my change.